crates/redeem/src/lib.rs

```diff
@@ -79,6 +79,9 @@ decl_storage! {
         /// risk the bitcoin client to reject the payment
         RedeemBtcDustValue get(fn redeem_btc_dust_value) config(): PolkaBTC<T>;
 
+        /// the expected size in bytes of the redeem bitcoin transfer
+        RedeemTransactionSize get(fn redeem_transaction_size) config(): u32 = 600u32;
```

Comment:
Ideally this default would be computed from constants that define the expected number of inputs / outputs.

Reply:
Can you set a computed default here?